server/spec/mutations/grid_services/create_spec.rb

```diff
@@ -352,6 +352,40 @@
       expect(outcome.success?).to be(false)
     end
 
+    it 'fails to save health_check, port 0' do
+      outcome = described_class.new(
+          grid: grid,
+          image: 'redis:2.8',
+          name: 'redis',
+          stateful: false,
+          health_check: {
+            protocol: 'tcp',
+            interval: 10,
+            timeout: 5,
+            initial_delay: 10,
+            port: 0
+          }
+      ).run
+      expect(outcome.success?).to be(false)
```

Comment:
General nitpick: I would suggest also checking the `expect(outcome.errors.message).to eq 'health_check' => ...`, otherwise this spec might not end up testing the health check validation, if it turns out to fail on some other parameter.